metabase/metabase
# setup rendering pipeline for echarts static pie

Part of https://github.com/metabase/metabase/issues/33281

### Description

Sets up the static viz rendering pipeline for our new static pie chart implementation using echarts. Currently it uses some mock values that will be replaced by the actual implementation in later PRs in this stack. It also currently adds a dynamic pie chart visualization type to the client, but this will also be removed later in the stack before it is all merged to master.

### How to verify

1. Create a question
2. Set visualization type to `Pie 2`
3. Add to a dashboard
4. Send in an email subscription
5. Confirm that a placeholder chart is rendered without any errors

### Demo

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/9ULpbAzGBtvQRyZEpGvu/8b5a9877-ccb4-48fa-905c-2f807e33153b/image.png)

^ output from subscription email

[Screen Recording 2023-10-16 at 2.00.12 PM.mov](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/9ULpbAzGBtvQRyZEpGvu/92243e9e-4481-4edd-a6b6-3abb6530db63/Screen%20Recording%202023-10-16%20at%202.00.12%20PM.mov)


### Checklist


_TODO: Need to update `js_svg_test.clj`_
